specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Trigger.json

```diff
@@ -525,7 +525,7 @@
         "offset": {
           "description": "Timespan applied to the start time of a tumbling window when evaluating dependency.",
           "type": "string",
-          "pattern": "((\\d+)\\.)?(\\d\\d):(60|([0-5][0-9])):(60|([0-5][0-9]))",
+          "pattern": "-((\\d+)\\.)?(\\d\\d):(60|([0-5][0-9])):(60|([0-5][0-9]))",
```

Comment:
Is there reason that one is `-?` and the other `-`?

Reply:
yes, the selfdependency can only have negative offset but a general dependency can have both positive and negative offset